doc/2/api/controllers/document/export/index.md

```diff
@@ -0,0 +1,128 @@
+---
+code: true
+type: page
+title: export
+---
+
+# export <SinceBadge version="auto-version"/>
+
+Export searched documents.
+
+This method behaves like a `document:search` except that it scrolls and formats the searched results in one of the supported format (`csv`, `jsonl`) then returns everything as an HTTP Stream,
+that can either be downloaded directly from a browser or any scripts.
+
+This method also supports the [Koncorde Filters DSL](/core/2/api/koncorde-filters-syntax) to match documents by passing the `lang` argument with the value `koncorde`.
+Koncorde filters will be translated into an Elasticsearch query.
+
+::: warning
+Koncorde `bool` operator and `regexp` clause are not supported for search queries.
+:::
+
+::: warning
+This method only supports the HTTP Protocol
+:::
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_export[?format=<export format>][&size=<int>][&scroll=<time to live>][&lang=<query language>]
+Method: GET
+Body:
+```
+
+```js
+{
+  "query": {
+    // ...
+  },
+  "aggregations": {
+    // ...
+  },
+  "sort": [
+    // ...
+  ],
+  "fields": [
+    // ...
+  ]
+}
+```
+
+You can also access this route with the `GET` verb:
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_export[?format=<export format>][&size=<int>][&scroll=<time to live>][&lang=<query language>]
+Method: GET
+```
+
+### Other protocols
+
+```js
+{
+  "index": "<index>",
+  "collection": "<collection>",
+  "controller": "document",
+  "action": "search",
+  "body": {
+    "query": {
+      // ...
+    },
+    "aggregations": {
+      // ...
+    },
+    "sort": [
+      // ...
+    ],
+    "fields": [
+    // ...
+    ]
+  },
+
+  // optional:
+  "size": <page size>,
+  "scroll": "<scroll duration>",
+  "lang": "<query language>",
+  "format": "<export format>"
+}
+```
+
+---
+
+## Arguments
+
+- `collection`: collection name
+- `index`: index name
+
+### Optionnal:
+
+- `separator`: This option is only supported for the `CSV` format, it defines which character sequence will be used to format the CSV documents
+- `scroll`: creates a forward-only result cursor. This option must be set with a [time duration](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/common-options.html#time-units), at the end of which the cursor is destroyed. If set, a cursor identifier named `scrollId` is returned in the results. This cursor can then be moved forward using the [scroll](/core/2/api/controllers/document/scroll) API action
+- `size`: set the maximum number of documents returned per result page. By default it's `10000`.
```

Comment:
Is it still used since we don't have pagination here?

Reply:
Yes it use internally, I will add a small paragraph that talks about thoses parameters and explains that are they used for.